src/styles/main.scss

```diff
@@ -0,0 +1,7 @@
+@import './index';
+@import './step';
+@import './title';
+@import './product';
+@import './star';
+@import './price';
+@import './button';
```

Comment:
You should use BEM classNaming
you should have only 2 block - card and stars

Reply:
What do you mean?